assets/docker/mssql/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.2'
 services:
-  mssql_db:
-    platform: linux/amd64
+  mssql-db:
+    platform: linux/x86_64
```

Comment:
Is there a way to parameterize this so it ups amd64 if we are using this locally on macs?